docs/monitoring/metrics.md

```diff
@@ -669,10 +670,11 @@ metrics.reporter.influxdb.port: 8086
 metrics.reporter.influxdb.db: flink
 metrics.reporter.influxdb.username: flink-metrics
 metrics.reporter.influxdb.password: qwerty
+metrics.reporter.influxdb.retentionPolicy: one_hr
```

Comment:
I know, this is a user-defined policy name, but how about using "one_hour" as in the tests?

Reply:
Updated.